bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_Zgamma_ZTobb_Mjj_200/VBF_Zgamma_ZTobb_Mjj_200_run_card.dat

```diff
@@ -0,0 +1,267 @@
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*********************************************************************
+   False  = cut_decays    ! Cut decay products 
+#*********************************************************************
+# Standard Cuts                                                      *
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+ 20.0  = ptj       ! minimum pt for the jets 
+ 20.0  = ptb       ! minimum pt for the b 
+ 20.0  = pta       ! minimum pt for the photons 
+ 10.0  = ptl       ! minimum pt for the charged leptons 
+ 0.0  = misset    ! minimum missing Et (sum of neutrino's momenta)
+ -1.0  = ptjmax    ! maximum pt for the jets
+ -1.0  = ptbmax    ! maximum pt for the b
+ -1.0  = ptamax    ! maximum pt for the photons
+ -1.0  = ptlmax    ! maximum pt for the charged leptons
+ -1.0  = missetmax ! maximum missing Et (sum of neutrino's momenta)
+#*********************************************************************
+# Minimum and maximum E's (in the center of mass frame)              *
+#*********************************************************************
+  0.0  = ej     ! minimum E for the jets 
+  0.0  = eb     ! minimum E for the b 
+  0.0  = ea     ! minimum E for the photons 
+  0.0  = el     ! minimum E for the charged leptons 
+  -1.0   = ejmax ! maximum E for the jets
+ -1.0   = ebmax ! maximum E for the b
+ -1.0   = eamax ! maximum E for the photons
+ -1.0   = elmax ! maximum E for the charged leptons
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+  6.0 = etaj    ! max rap for the jets 
+  -1.0  = etab    ! max rap for the b
+ 3.0  = etaa    ! max rap for the photons 
+ 3.0  = etal    ! max rap for the charged leptons 
+ 0.0  = etajmin ! min rap for the jets
+ 0.0  = etabmin ! min rap for the b
+ 0.0  = etaamin ! min rap for the photons
+ 0.0  = etalmin ! main rap for the charged leptons
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+ 0.4 = drjj    ! min distance between jets 
+ 0.0   = drbb    ! min distance between b's 
+ 0.4 = drll    ! min distance between leptons 
+ 0.4 = draa    ! min distance between gammas 
+ 0.0   = drbj    ! min distance between b and jet 
+ 0.4 = draj    ! min distance between gamma and jet 
+ 0.0 = drjl    ! min distance between jet and lepton 
+ 0.4   = drab    ! min distance between gamma and b 
+ 0.0   = drbl    ! min distance between b and lepton 
+ 0.4 = dral    ! min distance between gamma and lepton 
```

Comment:
For leptons this is irrelevant, but why are the cuts different for b jets and light jets? Is this on purpose?

Reply:
yes delta R for leptons are not usefull so should I remove the lines corresponding to the leptons. But for b-jets and q-jets which cuts are different I mean I imposed same deltaR cuts between the photon and b-jets and also between photon and q-jets. I got the clue from here: https://github.com/Soumyatifr/genproductions/blob/mg260legacy/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VBF_HH/VBF_HH_CV_1_C2V_1_C3_1_13TeV-madgraph/VBF_HH_CV_1_C2V_1_C3_1_13TeV-madgraph_run_card.dat#L143-L163 .
Please let me know if I missed something.

Reply:
@agrohsje please check if you have further comments.